packages-exp/auth-exp/src/model/auth.ts

```diff
@@ -48,6 +48,8 @@ export interface ConfigInternal extends Config {
   clientPlatform: ClientPlatform;
 }
 
+export type RefreshWithCustomToken = () => Promise<string | null>;
```

Comment:
CustomTokenProvider?

Reply:
Yeah, good q! So I wanted to distinguish it from the CustomTokenProvider itself - this refers to the wrapper around the CustomTokenProvider that also calls the signIn endpoint and updates the stsTokenManager